CMakeLists.txt

```diff
@@ -26,26 +31,27 @@ endif()
 
 add_compile_options(-Wall -Wextra -Wpedantic)
 include_directories(include)
-
-set(SOURCES src/inspection.cpp src/utils.cpp src/sparse_mesh.cpp)
+include_directories(${CMAKE_CURRENT_BINARY_DIR}) #ToDo probably better to have all .proto file in one folder and inlcude that instead of ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
I have not been able to get it to work with the dense.proto file in the include/vinspect path. I am not sure if this current solution is okay or if including the whole ${CMAKE_CURRENT_BINARY_DIR} is too much for just the resulting dense.pb.h file.

Reply:
I think for the moment it is fine, because the number of files is very low.